content/shmem_reductions.tex

```diff
@@ -206,24 +202,27 @@
     on all \acp{PE} in the active set are not still in use from a prior call to a
     collective \openshmem routine.  The \dest{} array on all \acp{PE} in the
     active set is ready to accept the results of the \OPR{reduction}.
-    
+
     Upon return from a reduction routine, the following are true for the local
-    \ac{PE}: The \dest{} array is updated and the \source{} array may be safely reused.  
+    \ac{PE}: The \dest{} array is updated and the \source{} array may be safely reused.
     The values in the \VAR{pSync} array are
     restored to the original values.
 
-    The sum and product reduction routines include complex-typed interfaces
-    for the \Cstd API only.
-    When the \Cstd translation environment does not support complex types%
-    \footnote{That is, under \Cstd language standards prior to \Cstd[99] or
-      under \Cstd[11] when \CONST{\_\_STDC\_NO\_COMPLEX\_\_} is defined to 1},
-    an \openshmem implementation is not required to provide support for
-    these complex-typed interfaces.
+    The sum and product reduction routines include complex-typed
+    interfaces for the \Cstd API only. When the \Cstd translation
```

Comment:
Should this be "C **and Fortran** API only"?

Reply:
Instead, I think the preceding line should read:

> The complex-typed interfaces are only provided for sum and product reductions.

Then, continue with "When the C translation environment...".

Reply:
👍 This is better.